tests/integration/components/partial-move-input-test.js

```diff
@@ -0,0 +1,35 @@
+import { test, moduleForComponent } from 'ember-qunit';
```

Comment:
so my comments above were about checking the partial_move screen but now I see you are testing the details component here instead - a much better idea!

Can you inject any mock data into the partial move component test so we can check for line items i.e. move qty X  from package Y in location Z and move qty A from package B in location C